tests/conftest.py

```
@@ -0,0 +1,379 @@
+# Test locations
+TEST_LOCATIONS = {
+    "RYSY_MOUNTAIN": {
+        "lat": 49.179,
+        "lon": 20.088,
+        "tile_url": "https://c.tile.openstreetmap.org/16/36424/22456.png",
+    },
+    "WROCLAW_CENTER": {"lat": 51.10655, "lon": 17.0555},
+}
+
+
+def pytest_configure(config):
```

Ruff (0.14.11)

Unused function argument: `config`

(ARG001)